Model/CategoryData.php

```diff
@@ -12,17 +12,24 @@ public function __construct(
         $this->storeManager = $storeManager;
     }
 
-    public function getCategories($storeId)
+    public function getCategoryQuery($storeId)
     {
-        $storeBaseUrl = $this->storeManager->getStore($storeId)->getBaseUrl(); // Used for multiwebsite configuration base url
-        $categoriesArray = [];
-        $categories = $this->categoryCollection->create()->addAttributeToSelect('name')
+        return $this->categoryCollection->create()->addAttributeToSelect('name')
                     ->joinTable(
                         ['url' => 'url_rewrite'],
                         'entity_id = entity_id',
                         ['request_path', 'store_id'],
                         ['entity_type' => 'category', 'store_id' => $storeId]
                     );
+    }
+
+    public function getCategories($storeId, $chunkId, $chunkItems)
```

Comment:
This comment is about all of the `Data` classes - product, order, category, customer:
Is the `$chunkItems` parameter ever different?
If not - why do you pass it half of the time and not just get it as a constant from the Import class in the `getXXX` function?